cmd/lakefs/cmd/setup.go

```diff
@@ -15,9 +15,9 @@ import (
 	"github.com/treeverse/lakefs/stats"
 )
 
-// initCmd represents the init command
-var initCmd = &cobra.Command{
-	Use:   "init",
+// setupCmd represents the setup command
```

Comment:
Please additionally keep the old "init" command as an alias for "setup", in case someone is already using it - but marked "deprecated", of course.

Reply:
Done. Cobra does have a `#MarkDeprecated` method but it doesn't appear to do anything (https://github.com/spf13/cobra/issues/463), but I included it in case it's fixed at some point.